js/widgets/widget.theme.js

```diff
@@ -29,17 +29,30 @@
 
 $.mobile.widget.theme = {
 	_create: function() {
+		var that = this;
 		this._super();
 		if ( !this.options.enhanced && this.options.theme ) {
-			this.widget().addClass( "ui-theme-" + this.options.theme );
+			$.each( this._themeElements(), function( i, toTheme ) {
+				that._addClass(
+					toTheme.element,
+					null,
+					toTheme.prefix + that.options[ toTheme.option || "theme" ]
+				);
+			} );
 		}
 	},
 
 	_setOption: function( key, value ) {
+		var that = this;
 		if ( key === "theme" && this.options.theme ) {
```

Comment:
This means this function only handles the theme option.

You should first retrieve this._themeElements(), loop over each member, and see if the option property is the one in the "key" param.

```JS
$.each( this._themeElements(), function( i, toTheme ) {
  var themeOption = ( toTheme.option || "theme" );
  if ( themeOption === key ) {
    that._removeClass(
      toTheme.element,
      null,
      toTheme.prefix + that.options[ themeOption ] )
    ._addClass( toTheme.element, null, toTheme.prefix + value );
  }
} );